.github/workflows/ci.yaml

```
@@ -11,14 +11,24 @@ jobs:
-      - name: Build and test
+      - name: Build
         run: yarn run build
 
-      # - name: Run tests
-      #   run: yarn test
+      - name: Create Cypress env file
+        run: |
+          echo '{"api_key":"${{ secrets.CYPRESS_API_KEY }}"}' > cypress.env.json
+
+      - name: Start Server
+        run:
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)